tests/e2e/main.go

```diff
@@ -240,10 +240,70 @@ var rootCmd = &cobra.Command{
 		}
 
 		// create status CRD
-		if statusCRD, err := ioutil.ReadFile(filepath.Join(assetsPath, manifestsFolder, "status-crd.yaml")); err != nil {
+		if CRDBytes, err := ioutil.ReadFile(filepath.Join(assetsPath, manifestsFolder, "status-crd.yaml")); err != nil {
```

Comment:
Would prefer crdBytes

Reply:
We use consistently upper case 
 for acronyms https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
ok